great_expectations/data_context/util.py

```diff
@@ -283,17 +283,7 @@ def default_checkpoints_exist(directory_path: str) -> bool:
         directory_path,
         DataContextConfigDefaults.DEFAULT_CHECKPOINT_STORE_BASE_DIRECTORY_RELATIVE_NAME.value,
     )
-    return (
-        os.path.isdir(checkpoints_directory_path)
-        and len(
-            [
-                os.path.join(checkpoints_directory_path, filename)
-                for filename in os.listdir(checkpoints_directory_path)
-                if filename.endswith(r".yml")
-            ]
-        )
-        > 0
-    )
+    return os.path.isdir(checkpoints_directory_path)
```

Comment:
@roblim I think that we need to bring back the full comprehensive check (i.e., not just that the directory is there but also that it has at least one YML file) -- the upgrader logic relies on this.

Reply:
I updated the upgrader logic to account for this change. I removed the second constraint to allow users with an existing project but empty checkpoints dir to still create checkpoint using `great_expectations checkpoint new`.